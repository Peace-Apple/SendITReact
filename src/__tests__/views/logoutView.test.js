import React from 'react';
import { shallow, mount } from 'enzyme';
import Logout from '../../views/logoutView';

const props = {
  history: {
    push: jest.fn(),
  },
};

describe('logout view', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Logout {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
