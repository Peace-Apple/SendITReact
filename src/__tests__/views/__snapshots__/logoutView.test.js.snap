// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logout view should render without crashing 1`] = `ShallowWrapper {}`;
